test/test-utils/createTestRenderer.js

```diff
@@ -0,0 +1,10 @@
+import React from "react";
```

Comment:
This is a test util I use for testing react components, example of usage can be seen in `App.test.js`